src/test/service/GeocodingService.test.ts

```diff
@@ -7,9 +7,11 @@ describe('tests for Geocoding Service', function () {
   let geocodingService: GeocodingService;
 
   beforeAll(() => {
-    geocodingService = new GeocodingService(
-      BaseController.createMockNodeGeocoder()
-    );
+    const mockGeocoder = BaseController.createMockNodeGeocoder();
+    const geocoderConfig = {
+      geocoder: mockGeocoder,
+    };
+    geocodingService = new GeocodingService(geocoderConfig);
```

Comment:
And then this can be more succinct
```suggestion
    geocodingService = new GeocodingService(
      BaseController.createMockNodeGeocoderConfig()
    );
```